csunplugged/templates/plugging_it_in/programming-challenge.html

```diff
@@ -56,14 +56,14 @@ <h5>{{ programming_challenge_item.challenge_set_number }}.{{ programming_challen
               <div class='col-2 d-flex align-items-center justify-content-center'>
               {% if programming_challenge_item.slug in saved_attempts %}
                 {% if saved_attempts|get_item:programming_challenge_item.slug|get_item:"status" == "failed" %}
-                  <p class="status-badge status-badge-incorrect">Incorrect</p> 
+                  <p class="status-badge bg-danger border-danger text-white">{% trans "Incorrect" %}</p> 
```

Comment:
_Might_ not need the `text-white`? Try it and see?

Reply:
I tried it earlier it kept it as the darker text, so think it might be needed.